badger/cmd/info.go

```diff
@@ -89,14 +97,20 @@ to the Dgraph team.
 
 func handleInfo(cmd *cobra.Command, args []string) error {
 	cvMode := checksumVerificationMode(opt.checksumVerificationMode)
+	ct, err := strToChecksumAlgorithm(opt.checksumAlgorithm)
+	if err != nil {
```

Comment:
there is no need for the if here, you could directly call y.Check(err) and it internally fails only when err is not nil.